plugins/woocommerce-admin/client/marketplace/components/my-subscriptions/my-subscriptions.scss

```diff
@@ -126,6 +126,14 @@ a.woocommerce-marketplace__refresh-subscriptions {
 	padding: 0 12px;
 }
 
+.woocommerce-marketplace__my-subscriptions .woocommerce-table.is-empty {
+	background: none;
+	border: 1px solid var(--lt-color-border-light);
```

Comment:
On my display I can barely see this color. Designs say `$gray-100` but that gives me the wrong value. It should be `#f0f0f0` but SCSS turns it into `#101517`. That is odd. Can we reference `##f0f0f0` directly here?

Reply:
Yap - same here. I see we're using below `border: 1px solid var(--gutenberg-gray-100, #f0f0f0);`

Reply:
Thanks! The variables are changing again and caused a little confusion. Thanks for spotting this, though. I've amended it to align with Lae's suggestion.